src/main/scala/org/broadinstitute/dsde/firecloud/utils/EnabledUserDirectives.scala

```diff
@@ -25,6 +26,8 @@ trait EnabledUserDirectives extends LazyLogging with SprayJsonSupport with Statu
   implicit val executionContext: ExecutionContext
   implicit val materializer: Materializer
 
+  val samDao: SamDAO
```

Comment:
changes in this file are related to moving the `getUserStatusInfo` call to samDAO